Set of changes to get Polish names working

jamie-dainton opened the pull request:
I was asked by a friend to see if I could get this working for Polish names and this is a quick run over,

1. It now picks up the active directory
2. I have renamed the directories to match the name in Rimworld
3. It now deletes the namebank to remove what might be there

It's not perfect but it met their requirements of having more Polish names in the game

AaronCRobinson commented:
Merged and updated to 1.3. Let me know if you prefer to have the changes supported for an older version and I can do a release for you. 

https://github.com/AaronCRobinson/NamesGalore/releases/tag/v1.3.0

I added a setting to toggle the change you added with clearing the name bank. If I did this wrong, please let me know. 

Sorry it took awhile. I don't really do much coding anymore.

jamie-dainton commented:
Thank you.

I'd forgotten all about this. I was on holiday and a friend's son loved
Rimworld and had an interest in programming so we did this as an evening
project.

On Fri, 20 Aug 2021 at 10:45, Aaron C Robinson ***@***.***>
wrote:

> Merged and updated to 1.3. Let me know if you prefer to have the changes
> supported for an older version and I can do a release for you.
>
> https://github.com/AaronCRobinson/NamesGalore/releases/tag/v1.3.0
>
> I added a setting to toggle the change you added with clearing the name
> bank. If I did this wrong, please let me know.
>
> Sorry it took awhile. I don't really do much coding anymore.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AaronCRobinson/NamesGalore/pull/5#issuecomment-902572825>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJLS24ZNQPJV2DSLL2NKUTT5YP23ANCNFSM4PL4RTTQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
